demos/segmentation_demo/python/segmentation_demo.py

```diff
@@ -71,10 +71,14 @@ def __init__(self, colors_path=None):
 
     def get_palette_from_file(self, colors_path):
         with open(colors_path, 'r') as file:
-            return [eval(line.strip()) for line in file.readlines()]
+            colors = []
+            for line in file.readlines():
+                values = line[line.index('(')+1:line.index(')')].split(',')
+                colors.append([int(v.strip()) for v in values])
```

Comment:
what if colors are invalid?
Should we handle and provide error message for this case?

Reply:
@eaidova do you mean validate that 'values' should be a number in 0...255 range?

Reply:
yes and not only:
* string can contains any other symbols which can not be casted to int
* list of values for one color can be not equal number of channels or even be empty

Reply:
I am not sure it is necessary. The README provides the fully description of correct file with colors. Also the ValueError occurs then some string has wrong format. There are many ways how invalid colors might look, even the replacement of round brackets by `[ ]` will lead to the error, so the handling of all this cases might make the code more complex.

Reply:
ok